state/saveAndRestore.go

```diff
@@ -892,45 +874,41 @@ func (ss *SaveState) MarshalBinary() (rval []byte, err error) {
 	if err != nil {
 		return nil, err
 	}
-	err = buf.PushUInt32(ss.EntryBlockDBHeightProcessing)
+
+	// formerly EntryBlockDBHeightProcessing, deprecated
+	err = buf.PushUInt32(0)
```

Comment:
could you leverage save state versioning to just completely get rid of those unused data? it means nodes will need to rebuild their state from 0, but that's not too bad and it's acceptable for Wax which is kind of a "new start" release

Reply:
Basically bump SaveStateVersion to 14?